semgrep_output_v1.atd

```diff
@@ -71,7 +71,7 @@ type raw_json <ocaml module="Yojson.Basic" t="t"> = abstract
 (*****************************************************************************)
 
 (* File path. less: could convert directly to Path class of pathlib library for Python *)
-type fpath <ocaml attr="deriving show"> <python decorator="dataclass(frozen=True)">= string wrap <ocaml module="ATD_string_wrap.Fpath">
+type fpath <ocaml attr="deriving show, eq, yojson"> <python decorator="dataclass(frozen=True)">= string wrap <ocaml module="ATD_string_wrap.Fpath">
```

Comment:
Why do you need yojson? In theory if you need to generate json strings from those types you can use
the generated string_of_xxx

Reply:
Turns out that it was unnecessary, removed!